src/System.Drawing.Common/src/System/Drawing/Gdiplus.cs

```diff
@@ -627,13 +533,8 @@ public static IntPtr CreateDIBSection(HandleRef hdc, ref NativeMethods.BITMAPINF
         [DllImport(ExternDll.Kernel32, SetLastError = true, ExactSpelling = true, CharSet = CharSet.Auto)]
         public static extern bool GlobalUnlock(HandleRef handle);
 
-        [DllImport(ExternDll.Gdi32, SetLastError = true, ExactSpelling = true, EntryPoint = "CreateRectRgn", CharSet = CharSet.Auto)]
-        private static extern IntPtr IntCreateRectRgn(int x1, int y1, int x2, int y2);
-
-        public static IntPtr CreateRectRgn(int x1, int y1, int x2, int y2)
-        {
-            return System.Internal.HandleCollector.Add(IntCreateRectRgn(x1, y1, x2, y2), SafeNativeMethods.CommonHandles.GDI);
-        }
+        [DllImport(ExternDll.Gdi32, SetLastError = true, ExactSpelling = true)]
```

Comment:
Does the GDI32 library have the same issues as GDI+? If it doesn't, then this should be fine.

Reply:
It doesn't